code/modules/uplink/uplink_items.dm

```diff
@@ -2230,11 +2230,11 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 
 /datum/uplink_item/role_restricted/pressure_mod
 	name = "Kinetic Accelerator Pressure Mod"
-	desc = "A modification kit which allows Kinetic Accelerators to do greatly increased damage while indoors and in a vacuum. \
+	desc = "A modification kit which allows Kinetic Accelerators to do greatly increased damage while indoors. \
 			Occupies 35% mod capacity."
 	item = /obj/item/borg/upgrade/modkit/indoors
-	cost = 5 //you need two for full damage, so total of 10 for maximum damage
-	limited_stock = 2 //you can't use more than two!
+	cost = 10 //you need one for full damage, so total of 10 for maximum damage
```

Comment:
I don't see anywhere where you buffed the pressure reduction module.
If we are nerfing through KPA to do reasonable damage in space, then wouldn't this now be worth less as opposed to more? Compare this to other items of similar value

Reply:
It'll only be worth for indoors, would 5 TC be okay?